### PR TITLE
Feature/328 333 dt role allign oneagent classic

### LINKED ISSUE #328: [FEATURE] Align dt* roles to actual functionality and responsibility
**Is your feature request related to a problem? Please describe.**
At the moment we have two dynatrace-related roles:
1. `dt-activegate`: installs a synthetic-only activegate
2. `dt-oneagent`: installs the dynatrace operator (dynakube) which also includes an activegate for kubernetes monitoring

The names for these roles currently do not represent what they actually do and are responsible for.

**Describe the solution you'd like**
Change the names as follows:
1. `dt-activegate` => `dt-private-synthetic-activegate`
2. `dt-oneagent` => `dt-operator`

In the future, we will get additional roles potentially (like `dt-oneagent-classic`, `dt-private-synthetic-kube`, ...)

### LINKED ISSUE #333: [FEATURE] OneAgent Classic Role
**Is your feature request related to a problem? Please describe.**
The current operator role is only suitable for kubernetes environments. We also need a role for non-k8s environments to deploy the oneagent.

### PR DESCRIPTION
**Pull request by kristofre**
Fixes #328 and #333

### CODE REVIEW

File: user-skel/.ace/ace [hunk 1]
```diff
@@ -21,8 +21,9 @@ install_targets = [
     'k8s',
     'gitea',
     'gitlab',
-    'dt_oneagent',
-    'dt_activegate',
+    'dt_operator',
+    'dt_private_synth_activegate_classic',
```
Comment (eemrdog) [inline user-skel/.ace/ace]: may be dt_activegate_private_synth_classic as naming preference to focus first that it is an activegate and then the option for this activegate which is for private_synth_classic, then later dt_activegate_private_synth_k8s after containerized version is launched

File: user-skel/ansible/roles/app-easytravel/Readme.md [hunk 2]
```diff
@@ -34,14 +34,14 @@ easytravel_image_tag: "2.0.0.3356" #image tag to deploy for all EasyTravel image
...
-    name: dt-oneagent
+    name: dt-operator
 ```
 
 ### (Optional) To install Dynatrace Activegate to enable synthetic monitoring
 
 ```yaml
 - include_role:
-    name: dt-activegate
+    name: dt-private-synth-activegate-classic
```
Comment (eemrdog) [inline user-skel/ansible/roles/app-easytravel/Readme.md]: same naming convention as mentioned above i.e. dt-activegate-private-synth-classic

**Review by eemrdog — COMMENTED** [review]
the only thing I commented is regarding the naming convention of activegate role. The rest seems ok to me

File: user-skel/.ace/ace (hunk 1 above)
Comment (kristofre) [reply user-skel/.ace/ace]: good point, pushed new changes to reflect that

**Review by eemrdog — APPROVED** [review]

**Pull request merged**